lib/convection/model/cloudfile.rb

```diff
@@ -61,8 +63,22 @@ def initialize(cloudfile)
         @stacks = {}
         @deck = []
         @stack_groups = {}
+        @thread_count = thread_count || 2
 
         instance_eval(IO.read(cloudfile), cloudfile, 1)
+
+        work_q = Queue.new
+        @deck.each { |stack| work_q.push stack }
+        workers = (0...@thread_count).map do
+          Thread.new do
+            until work_q.empty?
+              stack = work_q.pop(true)
+              stack.status
+              stack.template_info if stack.exist?
+            end
+          end
+        end
+        workers.map(&:join)
```

Comment:
Should we use `Enumerable#each` here since we don't care about the resulting array?